activemodel/lib/active_model/attribute_set.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-require "active_record/attribute_set/builder"
-require "active_record/attribute_set/yaml_encoder"
+require_relative "attribute_set/builder"
```

Comment:
We avoid using require_relative inside lib because it has problems with symlinked gems.